### PR TITLE
Update: JTL.Wawi version 1.5.47.2

### PR DESCRIPTION
**Pull request by DasCanard**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21948)

### CODE REVIEW

File: manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+PackageIdentifier: JTL.Wawi
+PackageVersion: 1.5.47.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
```
Comment (OfficialEsco) [inline manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml:12]: You do know the InstallerLocale is actually de-DE right? 🤔😁

File: manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  Architecture: neutral
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://downloads.jtl-software.de/setup-jtl-wawi_1.5.47.2_0719-0928_6a264dcd904.exe
+  InstallerSha256: 4AB1BCBA0C687BAF27C59BCA9C88A754822C0580866F87B784312AD2625D754A
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
```
Comment (OfficialEsco) [inline manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml:20]: The InstallerSwitches is not needed as they get auto put in when the InstallerType is Inno/Nullsoft etc

**Review by OfficialEsco — APPROVED** [review]

File: manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml (hunk 2 above)
Comment (DasCanard) [reply manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml:20]: Oh okay

File: manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml (hunk 1 above)
Comment (DasCanard) [reply manifests/j/JTL/Wawi/1.5.47.2/JTL.Wawi.installer.yaml:12]: Yeah i am going to fix it in the next Version

**Pull request merged**